e2e/rdctl.e2e.spec.ts

```diff
@@ -222,13 +222,16 @@ test.describe('Command server', () => {
     let resp = await doRequest('/v0/settings');
     const settings = await resp.json();
     const desiredEnabled = !settings.kubernetes.enabled;
-    const desiredEngine = settings.kubernetes.containerEngine === 'moby' ? 'containerd' : 'moby';
+    const desiredEngine = settings.containerEngine.name === 'moby' ? 'containerd' : 'moby';
     const desiredVersion = /1.23.4/.test(settings.kubernetes.version) ? 'v1.19.1' : 'v1.23.4';
     const requestedSettings = _.merge({}, settings, {
+      containerEngine: {
+        name:           { desiredEngine },
+        imageAllowList: { locked: !settings.containerEngine.imageAllowList.locked },
+      },
       kubernetes:
         {
           enabled:                    desiredEnabled,
-          containerEngine:            desiredEngine,
           version:                    desiredVersion,
           checkForExistingKimBuilder: !settings.kubernetes.checkForExistingKimBuilder, // not supported
```

Comment:
I thought all references to `kim` are supposed to be removed by this PR?

```console
$ agi kimbuild
e2e/rdctl.e2e.spec.ts
236:          checkForExistingKimBuilder: !settings.kubernetes.checkForExistingKimBuilder, // not supported

pkg/rancher-desktop/config/settings.ts
399:    // settings.kubernetes.checkForExistingKimBuilder = true;

pkg/rancher-desktop/main/commandServer/__tests__/settingsValidator.spec.ts
67:        ['kubernetes', 'checkForExistingKimBuilder'],

scripts/check-api-schema.ts
58:  'kubernetes.checkForExistingKimBuilder',
```

Reply:
good point...